contracts/sophia.md

```diff
@@ -546,6 +546,15 @@ The shortest unique suffix could also be used, and the hash could be ANDed with
 Then each exported function starts with an exported entry point where it executes
 code to fetch the arguments to memory.
 
+### Contract state
+
+Before a contract call is executed the current state of the contract is loaded
+into memory encoded as described above. A pointer to the encoded state is stored at
+address 0. The Sophia compiler generates code to unpack the encoded state and
```

Comment:
Maybe other addresses shall be documented? (I am thinking 32 and/or 64 based on skimming https://github.com/aeternity/epoch/pull/1097 )